pkg/db/migrations/001_job.up.sql

```diff
@@ -0,0 +1,27 @@
+CREATE TYPE job_result AS ENUM ('passed', 'failed', 'aborted');
+
+CREATE TABLE IF NOT EXISTS jobs (
```

Comment:
More a comment than a requested change, I'd typically use varchars for fields we know have finite limits and ones we'll want to potentially build indicies off of. 

Some of these fields are OCM fields and we can find out and duplicate their limits.

Reply:
Hmm. We certainly can do this. Do you have any links I can look at for the index-building implications of each option? I'm not familiar with all of that, though I know that the `text` type is stored differently than `varchar`.

Reply:
Did a quick look up to find some docs for you and found that in the background there's no functional difference (anymore?) between Text and varchar besides varchar allowing length-checking. 

I say ship it as is!